pyproject.toml

```diff
@@ -68,3 +68,8 @@ sphinx_rtd_theme = "*"
 [build-system]
 requires = ["setuptools","poetry>=0.12"]
 build-backend = "poetry.masonry.api"
+
+[tool.pytest.ini_options]
+filterwarnings = ["ignore:::cvxpy"]
```

Comment:
Hmm, I'm not sure if we want to ignore the CVXPY stuff. Should we keep these warnings in just in case they are actually useful?

Reply:
Most of the third party package deprecation warnings cannot be handled on our end. I feel it does clog up the pytest-cov output. 

The way I understand this, all the deprecations related to cvxpy dependencies will be ignored. But we should still be able to get the Deprecation warnings from cvxpy itself. I'll double check and link somethings I found later.

Reply:
Sounds good. For the  "too many subexpressions" it's arguable weather we want to solve them in this diff or not. If it looks like doing that would be a rabbit hole, I'd say just keep the warnings as we might be able to address this later in another diff.